src/main/java/com/amihaiemil/versioneye/RtRepository.java

```diff
@@ -116,4 +128,17 @@ public boolean fork() {
     public JsonObject json() {
         return this.repository;
     }
+
+    @Override
+    public void delete(final String branchName) throws IOException {
+        String repoKey = this.fullname().replace('.', '~').replace('/', ':');
+        this.req.uri()
+            .path(repoKey)
+            .queryParam(
+                "branch",
```

Comment:
as I see from documentation that ``replace()`` replaces all the occurences of the old char with the new one